code/modules/mob/new_player/new_player.dm

```diff
@@ -426,19 +428,9 @@
 	new_character.update_eyes()
 	new_character.regenerate_icons()
 	new_character.key = key//Manually transfer the key to log them in
-	if(new_character.client && new_character.client.prefs.has_soulcrypt)
+/*	if(new_character.client && new_character.client.prefs.has_soulcrypt)
 		new_character.create_soulcrypt()
-
-	//REEEEE GIMME A CRUCIFORM	-	Syzygy edit
-	var/datum/category_item/setup_option/core_implant/I = new_character.client.prefs.get_option("Core implant")	// gets the core implant selected in the character creator, which can either be the cruciform, or nothing.
-	if(I.implant_type)	//Checks if an implant is selected. If there is none selected, do nothing.
-		var/obj/item/weapon/implant/core_implant/C = new I.implant_type	//spawns the implant
-		C.install(new_character) //installs it inside you
-		C.activate() //turns it on
-		C.install_default_modules_by_job(new_character.mind.assigned_job)
-		C.access.Add(new_character.mind.assigned_job.cruciform_access)
-	//Best to fix the missing cruciform issue instead of relying on this crutch
-
+*/
```

Comment:
Massive chunk of our code got removed. Is this good? Could this be what was causing the implant issues?

Reply:
Naw, this one's the bandaid crap I slapped on. Now that the underlying issue is fixed, we don't need it anymore.